src/data/projects/newrelic-infra-identity-client-go.json

```diff
@@ -0,0 +1,26 @@
+{
+  "name": "infra-identity-client-go",
+  "fullName": "newrelic/infra-identity-client-go",
+  "slug": "newrelic-infra-identity-client-go",
+  "owner": {
+    "login": "newrelic",
+    "type": "Organization"
+  },
+  "title": "New Relic Infrastructure Identity - Go lang client",
+  "supportUrl": null,
+  "githubUrl": "https://github.com/newrelic/infra-identity-client-go",
+  "permalink": "https://opensource.newrelic.com/projects/newrelic/infra-identity-client-go",
+  "defaultBranch": "main",
+  "contributingGuideUrl": "https://github.com/newrelic/infra-identity-client-go/blob/main/CONTRIBUTING.md",
+  "iconUrl": "https://github.com/newrelic/infra-identity-client-go/blob/main/icon.png?raw=true",
+  "shortDescription": "Go auto-generated client for the OpenAPI Infrastructure Identity Client",
+  "description": "Go auto-generated client for the OpenAPI Infrastructure Identity Client",
+  "ossCategory": "new-relic-one-catalog-project",
```

Comment:
This appears to be the wrong categorization. I think you want `community-project` ([matching this slug](https://github.com/newrelic/opensource-website/blob/develop/src/data/oss-category/oss-category.json#L4)).